frameworks/svelte/__tests__/testUtils/testUtils.ts

```diff
@@ -3,20 +3,60 @@ import {render, RenderResult} from '@testing-library/svelte'
 import {testWithMockedIntersectionObserver} from '../../../../testUtils/setupIntersectionObserverMock';
 import type {SvelteComponentDev} from "svelte/internal";
 
+/**
+ * Get element from container
+ * @param container
+ * @param type
+ */
 const getElement = (container: HTMLElement, type: string): HTMLElement => container.querySelector(type) || {} as HTMLElement;
 
+/**
+ * Get img element from container
+ * @param container
+ */
 const getImageElement = (container: HTMLElement): HTMLImageElement => getElement(container, 'img') as HTMLImageElement;
 
-const getImageAttr = (container: HTMLElement, attr: string): any => ((getImageElement(container) || {}) as any)[attr];
-
+/**
+ * Get element attribute from container
+ * @param container
+ * @param type
+ * @param attr
+ */
 const getElementAttr = (container: HTMLElement, type: string, attr: string): any => ((getElement(container, type) || {}) as any)[attr];
```

Comment:
can we minimize the use of Any (consider that `any` also means `null`, `false`, `0`, `'foo'` and undefined)
Can we narrow these somehow?